cli/internal/cmd/info/bin.go

```diff
@@ -0,0 +1,77 @@
+package info
+
+import (
+	"errors"
+	"os"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"github.com/spf13/cobra"
+	"github.com/vercel/turborepo/cli/internal/cmdutil"
+	"github.com/vercel/turborepo/cli/internal/config"
+	"github.com/vercel/turborepo/cli/internal/logger"
+)
+
+type BinCommand struct {
+	Config *config.Config
+	Ui     *cli.ColoredUi
+}
+
+// Synopsis of run command
+func (c *BinCommand) Synopsis() string {
+	return "Get the path to the Turbo binary"
+}
+
+// Help returns information about the `bin` command
+func (c *BinCommand) Help() string {
+	helpText := `
+Usage: turbo bin
+
+  Get the path to the Turbo binary
+`
+	return strings.TrimSpace(helpText)
+}
+
+func (c *BinCommand) Run(args []string) int {
+	logger := logger.New()
+	cmd := BinCmd(&cmdutil.Helper{
+		Config: c.Config,
+		Logger: logger,
+	})
+
+	cmd.SilenceUsage = true
+	cmd.SilenceErrors = true
+	cmd.CompletionOptions.DisableDefaultCmd = true
+
+	err := cmd.Execute()
```

Comment:
Doesn't Cobra automatically take args through `os.Args` when calling `cmd.Execute()`?

Reply:
It does, but we support some root-level arguments too. So until we tie everything together, we want to scope it to just the arguments we're passing in. In fact, this might be a bug w/ `turbo login` too, as currently implemented. Try supplying `-v` or `-token` or any of the other arguments defined in `config.go`.